### DISCUSSION TITLE
Contributing to Revolt

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#282] [Announcements]
Moved to [CONTRIBUTING.md](https://github.com/revoltchat/.github/blob/master/.github/CONTRIBUTING.md).

**Comment by Bokipro360** [comment]
if this is contributing to revolt then i like to report a server Fsociety because that group is for hacking and it has so much hacking apps and programs, so i want semone to please read this and delete that server so members or semone else gets hacked.

**Reply by DeclanChidlow** [reply]
If you wish to report a server, please use the in-app reporting features so we can action it. Thanks!
